solana/pyth2wormhole/client/src/main.rs

```diff
@@ -438,29 +444,13 @@ async fn handle_attest_non_daemon_mode(
 ) -> Result<(), ErrBox> {
     let p2w_cfg = get_config_account(&lock_and_make_rpc(&rpc_cfg).await, &p2w_addr).await?;
 
-    // Use the mapping if specified
-    if let Some(mapping_addr) = attestation_cfg.mapping_addr.as_ref() {
-        match crawl_pyth_mapping(&lock_and_make_rpc(&rpc_cfg).await, mapping_addr).await {
-            Ok(additional_accounts) => {
-                debug!(
-                    "Crawled mapping {} data:\n{:#?}",
-                    mapping_addr, additional_accounts
-                );
-                attestation_cfg.add_symbols(additional_accounts, "mapping".to_owned());
-            }
-            // De-escalate crawling errors; A temporary failure to
-            // look up the mapping should not crash the attester
-            Err(e) => {
-                error!("Could not crawl mapping {}: {:?}", mapping_addr, e);
-            }
-        }
-    }
-    debug!(
-        "Attestation config (includes mapping accounts):\n{:#?}",
-        attestation_cfg
-    );
-
-    let batches = attestation_cfg.as_batches(p2w_cfg.max_batch_size as usize);
+    let batch_config =
+        attestation_config_to_batches(&rpc_cfg, &attestation_cfg, p2w_cfg.max_batch_size as usize)
+            .await?;
```

Comment:
I think screaming and trying to attest with what we got is better than failing the whole process here.